packages/components/nodes/chatmodels/ChatLocalAI/ChatLocalAI.ts

```diff
@@ -41,8 +52,10 @@ class ChatLocalAI_ChatModels implements INode {
             {
                 label: 'Model Name',
                 name: 'modelName',
-                type: 'string',
-                placeholder: 'gpt4all-lora-quantized.bin'
+                type: 'options',
```

Comment:
Thanks for the suggestion! We didn't use `options` because we didn't want to limit the selections, hence added as `string` so that user can just specify the model they have in LocalAI.

Other users might face confusion as in why there is no selection available unless they read the repo carefully and realize they have to specify `LOCALAI_CHAT_MODELS` env. For now, I'd prefer to switch back to `string`. In future, we will have community nodes which allow users to create their custom component - https://github.com/FlowiseAI/Flowise/pull/1066

Reply:
@HenryHengZJ Understood completely 👍🏻 , If I remove that and put it back to string are you ok with the way I implemented the auth with a credential? I'll fix up and remove that env var/option after work today when I get a chance.

Reply:
yep everything else looks good!

Reply:
and also you can fix the linting issues `yarn lint-fix`

Reply:
doh 🤦🏻 -  @HenryHengZJ ok I reverted what was discussed, and in my re-testing also figured out I needed to take a couple steps back and add back defaults for people not using keys. I think this should work with minimal risk/change to current users unless they want to turn on api auth on their LocalAI. 🤞🏻 Let me know if I need to change anything else or haven't followed your processes (apologies).

Reply:
awesome that all looks good! thanks!